### ISSUE TITLE
seriesStmt for SBD

### ISSUE THREAD

**Issue by dlschwartz** [srophe/srophe-app-data#1020]
Rather than transform existing persons data into the correct format, we have [removed thing that don't belong in the titleStmt](https://github.com/srophe/srophe-app-data/commit/e0505ca805171a9d6754377d845557f73a79b4c2). 

The next step will be to create the proper seriesStmt elements for each series (see below) based on [this drawing](url).

Related issues
- should we include "anonymous" as a series?
- should the app create Qadishe, Authors, etc. out of the body (i.e. person/@ana='#syriaca-author') or out of the new seriesStmt?

**Comment by dlschwartz** [comment]
Draft seriesStmt elements for persons records:

```
         <seriesStmt>
            <title level="s" xml:lang="en">The Syriac Biographical Dictionary</title>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#dmichelson">David A. Michelson</editor>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#jnsaint-laurent">Jeanne-Nicole Mellon Saint-Laurent</editor>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#dschwartz">Daniel L. Schwartz</editor>
            <editor role="associate" ref="http://syriaca.org/documentation/editors.xml#ngibson">Nathan P. Gibson</editor><!-- What @role should Nathan have? -->
            <idno type="URI">http://syriaca.org/persons</idno>
         </seriesStmt>
         <seriesStmt>
            <title level="s">Gateway to the Syriac Saints</title>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#jnsaint-laurent">Jeanne-Nicole Mellon Saint-Laurent</editor>
            <idno type="URI">http://syriaca.org/saints</idno>
         </seriesStmt>
         <seriesStmt>
            <title level="m">Qadishe: A Guide to the Syriac Saints</title>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#jnsaint-laurent">Jeanne-Nicole Mellon Saint-Laurent</editor>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#dmichelson">David A. Michelson</editor>
            <idno type="URI">http://syriaca.org/saints</idno>
         </seriesStmt>
         <seriesStmt>
            <title level="m">A Guide to Syriac Authors</title>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#ngibson">Nathan P. Gibson</editor>
            <editor role="general" ref="http://syriaca.org/documentation/editors.xml#dmichelson">David A. Michelson</editor>
            <idno type="URI">http://syriaca.org/authors</idno>
         </seriesStmt>
```

Issues:
- We need to make sure that the correct editors are listed and their role is defined correctly.
- N.B., I have removed tei:biblScope because the TEI uses it differently than we were in our persons records. [We have been using this](https://github.com/srophe/srophe-app-data/blob/master/data/persons/tei/13.xml#L135) to indicate the constituent parts of the parent::seriesStmt. The TEI uses biblScope, see [code examples](https://tei-c.org/release/doc/tei-p5-doc/en/html/examples-seriesStmt.html) and [guidelines](https://tei-c.org/release/doc/tei-p5-doc/en/html/HD.html#HD26) to indicate the position that the ancestor::titleStmt/title has with respect to the series mentioned in here in the seriesStmt.
- However, the problem we have then is that we don't describe in this location the relationship between the title level='s' entities and the title level='m' entities.

**Comment by dlschwartz** [comment]
For XQuery transform:
- All persons are part of SBD
- All saints (//tei:person/@ana='#syriaca-saint') are part of Qadishe and Gateway to the Syriac Saints.
- All authors (//tei:person/@ana='#syriaca-author') are part of Guide to Syriac Authors

**Comment by wlpotter** [comment]
Script for this is ready to go: https://github.com/srophe/srophe-xQueries/blob/master/sbd-schema-and-normalizations-2023/update-seriesStmt.xq

Waiting on final series statement. @dlschwartz let me know when the seriesStmt language is fully approved and ready to update in all records

**Comment by dlschwartz** [comment]
@wlpotter I updated the seriesStmt in the comment above. I have included you and Nathan (I hope correctly, please let me know if there's anything off).

**Comment by wlpotter** [comment]
The one thing I see is that I think the URI for Qadishe is http://syriaca.org/q

**Comment by dlschwartz** [comment]
Good catch! Updated. By the way, I'm doing this work in this branch: https://github.com/srophe/srophe-app-data/tree/2023.02.21_schemaWork

**Comment by wlpotter** [comment]
Pull request to bring these into your schema work branch is here for your review: https://github.com/srophe/srophe-app-data/pull/1032
